lib/metadata/wrapper.js

```diff
@@ -227,6 +227,18 @@ const metadata = {
         }
         return client.getDiskUsage(cb);
     },
+
+    countItems: (log, cb) => {
+        if (!client.countItems) {
+            log.debug('returning zero item counts as fallback', { implName });
```

Comment:
you'd rather return a response with 0 items instead of returning not implemented?